### PR TITLE
62 frp remove internal pages

### LINKED ISSUE #62: FRP: Remove internal pages
### What is your proposal 🚀
The internal pages should be removed thanks to ySync: https://sync.yearn.farm/

### What does this solve ? 🛠
Remove duplication and out-of-scope product

### More 🔎
Be careful to check all the stuff related to internal, maybe some stuff in API will need to be removed!

### PR DESCRIPTION
**Pull request by Majorfi**
## Description
Fixes #62 
- Removing internal pages
- Removing deprecated CSS styling to use Web-Lib style
- Enabling PWA
- Enabling IPFS deployment

## Type of change
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change details
*N/A*

### CODE REVIEW

File: components/Menu.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import	React, {ReactElement}				from	'react';
 import	{useRouter}			from	'next/router';
 import	Link				from	'next/link';
 import	useLocalization		from	'contexts/useLocalization';
-import	useUI				from	'contexts/useUI';
+import	{useUI}				from	'@yearn-finance/web-lib/contexts';
```
Comment (0xMirim) [inline components/Menu.tsx:5]: Replacing local useUI hook with with web-lib implementation causes the page to be displayed in dark theme for first time visitors. 

This is be reproducible by running the code locally and opening the page in a private window. This ensures that no previous theme is saved in localStorage (simulating first time visitor experience).

From what I've been able to gather the site always sets the theme to dark and saves this value in localStorage when no previous value exists (or when you manually remove the local storage value and reload the page). 

This even happens on the current production site, though it seems that the site ignores the dark theme that gets saved in localStorage and displays the light theme even to first time visitors (which I believe is the desired behavior). 

localStorage-theme-dark

This change causes the new version of the site to actually display the dark theme which is set by default. I wanted to point this out even though I haven't been able to figure out where this default 'dark' theme gets initially set on localStorage.

File: components/StandardFooter.tsx [hunk 2]
```diff
@@ -10,32 +10,32 @@ import meta from 'public/manifest.json';
 function	Footer(): ReactElement {
 	const	{common} = useLocalization();
 	return (
-		<footer className={'hidden flex-row items-center py-8 mx-auto w-full max-w-6xl bg-white-blue-1 dark:bg-black-1 md:flex'}>
+		<footer className={'mx-auto hidden w-full max-w-6xl flex-row items-center py-8 md:flex'}>
 			<Link href={'/disclaimer'}>
-				<p className={'pr-6 text-gray-blue-1 dark:text-gray-3 link'}>{'Disclaimer'}</p>
+				<p className={'link pr-6 text-neutral-700'}>{'Disclaimer'}</p>
```
Comment (0xMirim) [inline components/StandardFooter.tsx:15]: Nothing needs to be adjusted I wanted a share my opinion that replacing the current link underline with no-underline or an underline only on hover might be more consistent with majority of yearn sites.

**Review by 0xMirim — COMMENTED** [review]
You changes are good I just wanted to point out the couple of minor things I noticed.

**Comment by 0xMirim** [comment]
Quick addition CSS changes cause text on 404 page to not be readable in light theme.

**Review by 0xMirim — APPROVED** [review]
LGTM!

**Review by DarkGhost7 — APPROVED** [review]
lgtm

**Pull request merged**